governance/votes/Executive vote - October 04, 2024.md

```diff
@@ -0,0 +1,158 @@
+- Set up the D3M DC-IAM.
+- Set up the D3M addresses.
+- Extend the [DIRECT_MOM Breaker](https://sky-atlas.powerhouse.io/#A.1.8.2.2.5_Direct_Deposit_Breaker_Exception-c240d4fa-ab20-4787-8dbf-7c94c8e78347%7C0db30758e055352c) to include the new D3M.
+
+These actions are further described below.
+
+#### New ilk Addition
+
+A new ilk will be added as DIRECT-SPK-AAVE-LIDO-USDS.
+
+#### Aave Lido Market Spark USDS D3M Setup
+
+- [Maximum Debt Ceiling (`line`)](https://sky-atlas.powerhouse.io/#A.3.4.1.5.1.1.3_Maximum_Debt_Ceiling_Definition-e4a0d8f1-4a01-46a7-b693-a94e7f4afc1a|57ea599773d45150b2632df9): 100 million USDS.
+- [Target Available Debt (`gap`)](https://sky-atlas.powerhouse.io/#A.3.4.1.5.1.1.4_Target_Available_Debt_Definition-b876ecf4-a901-4721-9c4b-9f2fc21f954c|57ea599773d45150b2632df9): 50 million USDS.
+- [Ceiling Increase Cooldown (`ttl`)](https://sky-atlas.powerhouse.io/#A.3.4.1.5.1.1.5_Ceiling_Increase_Cooldown_Definition-3d5980c5-7036-4b68-93e0-a48551fe09ed|57ea599773d45150b2632df9): 24 hours.
+- Debt Write-off Timelock (`tau`) (Defines the time to elapse before debt can be written off after a D3M has been caged): 7 days.
+
+#### D3M Addresses Setup
+
+- oracle: D3MOracle
+- plan: [D3MOperatorPlan](https://github.com/makerdao/dss-direct-deposit/blob/master/src/plans/D3MOperatorPlan.sol)
+- pool: [D3MAaveV3USDSNoSupplyCapTypePool](https://github.com/makerdao/dss-direct-deposit/blob/master/src/pools/D3MAaveV3USDSNoSupplyCapTypePool.sol)
+- aToken: TBC
+- operator: [0x298b375f24CeDb45e936D7e21d6Eb05e344adFb5](https://etherscan.io/address/0x298b375f24cedb45e936d7e21d6eb05e344adfb5)
+- stabledebt address: TBC
+- variabledebt address: TBC
+
+#### DIRECT_MOM Breaker Extension
+
+If this executive proposal passes, then the [DIRECT_MOM Breaker](https://sky-atlas.powerhouse.io/#A.1.8.2.2.5_Direct_Deposit_Breaker_Exception-c240d4fa-ab20-4787-8dbf-7c94c8e78347%7C0db30758e055352c) will be extended to include the Aave Lido Market Spark USDS D3M, enabling governance to deactivate vault functionality through an executive spell without waiting for the GSM Pause Delay.
+
+### WBTC Risk Mitigation
+
+- **Authorization**: [Ecosystem Approval](https://forum.sky.money/t/wbtc-changes-and-risk-mitigation-10-august-2024/24844/52)
+- **Proposal**: [Forum Post](https://forum.sky.money/t/wbtc-changes-and-risk-mitigation-10-august-2024/24844/48)
+
+If this executive proposal passes, then the following WBTC risk mitigation actions will be executed on WBTC Legacy Vaults:
+
+- Decrease [Liquidation Penalty (`chop`)](https://sky-atlas.powerhouse.io/#A.3.8.1.1.2.5.7_Liquidation_Penalty_(chop)-5295aa1f-3944-4fde-aaec-d999244b57e9|57eaf45219bea3b430c2a59a) for WBTC-A, WBTC-B, and WBTC-C from 13% to **0%**.
+- Increase WBTC-A [Linear Interpolation (LERP)](https://manual.makerdao.com/module-index/module-lerp) for [Liquidation Ratios](https://manual.makerdao.com/parameter-index/vault-risk/param-liquidation-ratio) from 145% to **150%** over 6 days.
```

Comment:
note to @VoteWizard - let's ask @Le-Bateleur-2023 to get LERP added to the atlas.